packages/rn-tester/RNTester/NativeExampleViews/UpdatePropertiesExampleView.m

```diff
@@ -53,16 +53,18 @@ - (instancetype)initWithFrame:(CGRect)frame
                                          moduleName:@"SetPropertiesExampleApp"
                                   initialProperties:@{@"color" : @"beige"}];
 
-#if !TARGET_OS_OSX // [macOS]
+    // [macOS Github#1642: Suppress analyzer error of nonlocalized string
+    NSString *buttonTitle = NSLocalizedString(@"Native Button", nil);
+#if !TARGET_OS_OSX
     _button = [UIButton buttonWithType:UIButtonTypeRoundedRect];
-    [_button setTitle:@"Native Button" forState:UIControlStateNormal];
+    [_button setTitle:buttonTitle forState:UIControlStateNormal];
```

Comment:
Possible inline comment to help with future merges?

Reply:
Added!